app/helpers/template_helpers.py

```diff
@@ -119,20 +119,16 @@ def service_links_context(
     @property
     def data_layer_context(
         self,
-    ) -> list[dict]:
+    ) -> dict[str, str]:
         tx_id_context = (
             {"tx_id": metadata.tx_id}
             if (metadata := get_metadata(current_user))
-            else None
+            else {}
         )
-        additional_context = self._survey_config.get_additional_data_layer_context()
         schema_context = {
             key: value for key in DATA_LAYER_KEYS if (value := cookie_session.get(key))
         }
-        context = [*additional_context, schema_context]
-        if tx_id_context:
-            context.append(tx_id_context)
-        return context
+        return tx_id_context | schema_context
```

Comment:
Just a general Python-y question - is using `|` the preferred way that we merge dicts? I've seen it like `{**tx_id_context, **schema_context}` so just wanted to ask for my understanding.

Reply:
I think it is the most modern way of merging dicts, introduced in 3.9 so very recently.

Reply:
Ah awesome, thanks for the explanation 👍